tomcat-conf/server.xml

```diff
@@ -176,7 +176,44 @@
                errorCode.403="webapps/ROOT/error403.html"
                showReport="true"
                showServerInfo="false" />
-
+        <Context>
+          <!-- When including prconfig settings as JNDI environment entries, make sure that:
```

Comment:
perhaps clarify this a little and provide an example of what works...better to show success ;-)>>
<!-- To include JNDI environment entries in a customized prconfig setting, you must ensure that:
            1. JNDI names are unique.
            2. You do not include an existing JNDI name as a prefix to another JNDI name.
               For example you cannot use 'abc/feature1' and 'abc/feature1/threshold' to bind to objects; instead
               you could use "abc/<<what??- provide example>>' and 'abc/<<what??- provide example>>/threshold'
          -->